src/perl/Permabit/KernelModule.pm

```diff
@@ -63,11 +63,11 @@ sub load {
         $log->debug("kernel log additions:\n$additions");
       }
     };
+    $log->info("Using kernel module: " . $modName);
     $self->_step(command => "sudo modprobe $modName $self->{modprobeOption}",
                  cleaner => "sudo modprobe -r $modName",
                  diagnostic => $uponModprobeError);
   }
-
   return (!$machine->runSystemCmd("sudo modinfo $modName"));
```

Comment:
Don't remove this line, I like the extra space. (Generally a blank line is good after the end of a block.)